jdaviz/configs/default/plugins/model_fitting/model_fitting.vue

```diff
@@ -196,7 +196,7 @@
 
       <v-row>
         <span class="v-messages v-messages__message text--secondary">
-            If fit is not sufficiently converged, try clicking fitting again to complete additional iterations.
+            If fit is not sufficiently converged, click Fit Model again to run additional iterations.
```

Comment:
The change to this message wording isn't strictly necessary for this PR, but I found it helpful when learning to use this workflow.